src/PlaneWaveBasis.jl

```diff
@@ -138,6 +128,18 @@ function PlaneWaveBasis(model::Model{T}, Ecut::Number,
         kweights, ksymops, fft_size, grids, opFFT, ipFFT, opIFFT, ipIFFT
     )
 end
+function PlaneWaveBasis(model::Model{T}, Ecut::Number, kgrid_size::AbstractVector;
+                        bzmesh_symmetry=:full, kwargs...) where {T <: Real}
+    kcoords, ksymops = nothing, nothing
```

Comment:
Don't need that

Reply:
True, since when ... I always had in mind `if` opens a new scope ...

Reply:
I always get very confused by this so for more complicated cases it's probably worth it to be a bit more explicit, but in this case it's clear